<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrumentWindow</class>
 <widget class="QWidget" name="InstrumentWindow">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="listView_Available">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>291</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_Exist">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>291</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
